src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -579,7 +579,7 @@ private void ProcessMTUSize(string targetNameOrAddress)
                 WriteObject(new PingMtuStatus(
                     Source,
                     resolvedTargetName,
-                    replyResult ?? throw new ArgumentNullException(nameof(replyResult)),
+                    replyResult ?? throw new SystemException(nameof(replyResult)),
```

Comment:
I believe we shouldn't use SystemException because it is _generic base_ class.

Reply:
I can refactor this code to not throw, in new PR.

Reply:
`SystemException` is actually a reserved exception type, using it would be a violation of [CA2201](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/CA2201)